ansible_deploy/command_line.py

```diff
@@ -263,15 +279,74 @@ def setup_ansible(setup_hooks, commit):
         else:
             logger.error("Not supported")
 
-def run_task(config, options):
+
+def get_play_items(config: dict, task_name: str):
+    """
+    Function obtaining play items for specified task.
+    :param config:
+    :param task_name:
+    :return:
+    """
+    play_items = []
+
+    for item in config["tasks"]["tasks"]:
+        if item["name"] == task_name:
+            play_names = item["play_items"]
+
+    for play in play_names:
+        for item in config["play_items"]:
+            if item["name"] == play:
+                play_items.append(item["file"])
+
+    return play_items
+
+
+def run_task(config: dict, options: dict, inventory: str):
     """
     Function implementing actual execution of ansible-playbook
     """
+    play_items = get_play_items(config, options["task"])
+    for play in play_items:
+        command = ["ansible-playbook", "-l", options["infra"], "-i", inventory, play]
+        try:
+            with subprocess.Popen(command, stdout=subprocess.PIPE, stderr=subprocess.PIPE) as proc:
+                std_oe = proc.communicate()
+                for std in std_oe:
+                    for line in std.split(b"\n\n"):
+                        logger.info(line.decode("utf-8"))
+        except Exception as exc:
+            logger.error(exc)
```

Comment:
Error message with something specific to ansible-deploy to make it easy to find in the code like:
logger.error("Failed to run playbook:%s, Error was:%s", play, exc) ?
exit here on failure?